### PR TITLE
ddhcpd: Remove config defaults that won't work in other comunities

### PR DESCRIPTION
**Pull request by TobleMiner**
The upgrade script will take care of creating the missing entries with
values appropriate for the current community

### CODE REVIEW

File: ddhcpd/files/etc/config/ddhcpd [hunk 1]
```diff
@@ -6,19 +6,8 @@ config ddhcpd 'settings'
...
 
-config dhcp_option 'server_identify'
-     option code '54'
-     option len '4'
-     option payload '10.116.254.254'
-
-config dhcp_option 'netmask'
-     option code '1'
-     option len '4'
-     option payload '255.255.128.0'
```
Comment (rubo77) [inline ddhcpd/files/etc/config/ddhcpd]: `payload` is the value that is set by broadcast in site.conf?

Comment (TobleMiner) [reply ddhcpd/files/etc/config/ddhcpd]: Yes. All the options I removed are set up by https://github.com/sargon/gluon-sargon/blob/master/ddhcpd/luasrc/lib/gluon/upgrade/500-ddhcpd

I'd rather like ddhcpd not starting up at all than running with a config not appropriate for your community

Comment (rubo77) [reply ddhcpd/files/etc/config/ddhcpd]: why is the value `broadcast` just deleted then?

https://github.com/sargon/gluon-sargon/blob/e8930700f8d640dd7d3e4ed5258e859af854ee7c/ddhcpd/luasrc/lib/gluon/upgrade/500-ddhcpd#L21

Comment (rubo77) [reply ddhcpd/files/etc/config/ddhcpd]: I think it is not optional, but obsolete, it is used nowhere i can find

Comment (TobleMiner) [reply ddhcpd/files/etc/config/ddhcpd]: Yes-ish. It is used to remove this section from old configs that do still contain a broadcast section.

ddhcpd can actually interprete that section thus it is not obsolete.

Comment (rubo77) [reply ddhcpd/files/etc/config/ddhcpd]: OK, but where does it interpret `broadcast`?

**Pull request merged**